<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.10.0" xml:lang="en-US">
  <compounddef id="classutils_1_1exceptions_1_1InvalidMapData" kind="class" language="Python" prot="public">
    <compoundname>utils::exceptions::InvalidMapData</compoundname>
    <basecompoundref prot="public" virt="non-virtual">Exception</basecompoundref>
    <sectiondef kind="public-func">
      <memberdef kind="function" id="classutils_1_1exceptions_1_1InvalidMapData_1adc0bf2bb04cb608bd84ae8f76c8bc7a1" prot="public" static="no" const="no" explicit="no" inline="no" virt="non-virtual">
        <type>None</type>
        <definition> None utils.exceptions.InvalidMapData.__init__</definition>
        <argsstring>(self, message=&quot;Invalid map data, have you initialise the map with initialise()?&quot;)</argsstring>
        <name>__init__</name>
        <qualifiedname>utils.exceptions.InvalidMapData.__init__</qualifiedname>
        <param>
          <type>self</type>
          <defname>self</defname>
        </param>
        <param>
          <type>message</type>
          <defname>message</defname>
          <defval>&quot;Invalid map data, have you initialise the map with initialise()?&quot;</defval>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="/home/samuel/Documents/Python/Tile-Game/game/utils/exceptions.py" line="17" column="1" bodyfile="/home/samuel/Documents/Python/Tile-Game/game/utils/exceptions.py" bodystart="17" bodyend="20"/>
      </memberdef>
    </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
<para><verbatim>Class for creating the exception caused by invalid map data.
</verbatim> </para>
    </detaileddescription>
    <inheritancegraph>
      <node id="2">
        <label>Exception</label>
      </node>
      <node id="1">
        <label>utils.exceptions.InvalidMapData</label>
        <link refid="classutils_1_1exceptions_1_1InvalidMapData"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="2">
        <label>Exception</label>
      </node>
      <node id="1">
        <label>utils.exceptions.InvalidMapData</label>
        <link refid="classutils_1_1exceptions_1_1InvalidMapData"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
      </node>
    </collaborationgraph>
    <location file="/home/samuel/Documents/Python/Tile-Game/game/utils/exceptions.py" line="12" column="1" bodyfile="/home/samuel/Documents/Python/Tile-Game/game/utils/exceptions.py" bodystart="12" bodyend="20"/>
    <listofallmembers>
      <member refid="classutils_1_1exceptions_1_1InvalidMapData_1adc0bf2bb04cb608bd84ae8f76c8bc7a1" prot="public" virt="non-virtual"><scope>utils::exceptions::InvalidMapData</scope><name>__init__</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
